executor/poolmgr/gp.go

```diff
@@ -92,6 +86,7 @@ func MakeGenericPool(
 	namespace string,
 	functionNamespace string,
 	fsCache *fscache.FunctionServiceCache,
+	fetcherConfig *fetcherConfig.Config,
 	instanceId string,
 	enableIstio bool,
 	collectorEndpoint string) (*GenericPool, error) {
```

Comment:
Did you mean to remove `collectorEndpoint` from here?

Reply:
Yes, good catch